Assets/GothicVR/Scripts/Demo/LoadFollower.cs

```diff
@@ -0,0 +1,36 @@
+using UnityEngine;
+
+public class LogoFollower : MonoBehaviour
+{
+    public Transform target; // Reference to the main camera
+    public float smoothness = 0.1f; // Smoothing factor for the logo movement
+    public float followDelay = 0.5f; // Delay between the logo and the main camera
+
+    private Vector3 initialOffset; // Initial offset between the logo and the camera
+    private Vector3 velocity = Vector3.zero; // Velocity for smoothing the movement
+
+    private void Start()
+    {
+        // Calculate and store the initial offset between the logo and the camera
+        initialOffset = transform.position - target.position;
+    }
+
+    private void LateUpdate()
```

Comment:
Why is late Update needed? Please comment if so.

Reply:
This was so we can add a delay to when the gameobject this script was attached to (loading canvas) was moved based on the transform specified.
will add a summary to it if this will still be used